lsp/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ jlink {
   addExtraDependencies("jline-terminal-jansi")
   mergedModule {
     additive = true
+    requires("com.google.gson")
```

Comment:
🤯

Reply:
Guessed from the error message. Passed LSP initialization in my machine 😆